content/en/blog/_posts/2024-08-21-cgroup-driver-from-cri.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+title: "Configuring Kubernetes just got easier: cgroup driver from CRI"
+date: 2024-08-06
+slug: cgroup driver from CRI to beta
+author: >
+  Peter Hunt (Red Hat)
+---
+
+Historically, the cgroup driver has been a pain point for users running new
+Kubernetes clusters. On Linux systems, there are two different cgroup drivers:
+`cgroupfs` and `systemd`. In the past, both the [kubelet](/docs/reference/command-line-tools-reference/kubelet/)
+and CRI implementation (like CRI-O or containerd) needed to be configured to use
+the same cgroup driver, or else the kubelet would exit with an error. This was a
+source of headaches for many cluster admins. However, there is light at the end of the tunnel!
+
+## `KubeletCgroupDriverFromCRI` Feature Gate
+
+In v1.28.0, the SIG Node community introduced the feature gate
+`KubeletCgroupDriverFromCRI`, which instructs the kubelet to ask the CRI
+implementation which cgroup driver to use. It took a couple of releases for
+support to land in the major two CRI implementations (containerd and CRI-O), but as of v1.31.0, this
+feature is now beta!
+
+In addition to setting the feature gate, a cluster admin needs to ensure their
+CRI implementatikn is new enough:
```

Comment:
The current statement is likely to confuse users, as they might think they need to be on the latest version of the CRI for this to work. From the information given below, it would make more sense to say supported instead of new enough. 

```suggestion
CRI implementation is new enough:
```